<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditEntryWidgetMain</class>
 <widget class="QWidget" name="EditEntryWidgetMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>301</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::ExpandingFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Title:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="titleEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="usernameLabel">
     <property name="text">
      <string>Username:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="usernameEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="urlLabel">
     <property name="text">
      <string>URL:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="urlEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="passwordLabel">
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="passwordEdit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="togglePasswordButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="passwordRepeatLabel">
     <property name="text">
      <string>Repeat:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="passwordRepeatEdit">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="expireCheck">
     <property name="text">
      <string>Expires</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDateTimeEdit" name="expireDatePicker">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>titleEdit</tabstop>
  <tabstop>usernameEdit</tabstop>
  <tabstop>urlEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>passwordRepeatEdit</tabstop>
  <tabstop>togglePasswordButton</tabstop>
  <tabstop>expireCheck</tabstop>
  <tabstop>expireDatePicker</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
